### PR TITLE
Fixes #33468 - remove background download policy references

### PR DESCRIPTION
**Pull request by ianballou**
I think I've got them all, but help me check for other references that I might have missed.

Try changing your smart proxies and download policy settings to use background and then migrate.  They should all change to immediate.

### CODE REVIEW

File: app/controllers/katello/api/v2/products_bulk_actions_controller.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def sync_products
       syncable_roots = RootRepository.where(:product_id => syncable_products).has_url
 
       syncable_roots = syncable_roots.yum_type if skip_metadata_check || validate_contents
-      syncable_roots = syncable_roots.where.not(:download_policy => ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND) if validate_contents
+      syncable_roots = syncable_roots.where.not(:download_policy => 'on_demand') if validate_contents
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/products_bulk_actions_controller.rb:38]: Should we put these constants in somewhere like Katello::RootRepository ?

Comment (jlsherrill) [reply app/controllers/katello/api/v2/products_bulk_actions_controller.rb:38]: +1 to removing runcible references though!

Comment (ianballou) [reply app/controllers/katello/api/v2/products_bulk_actions_controller.rb:38]: Sure, I can do that.

**Review by jlsherrill — APPROVED** [review]
ACK, searched through and could find no other references

**Comment by ianballou** [comment]
[test katello]

**Comment by ianballou** [comment]
[test katello]

**Pull request merged**